src/runtime/pkg/katautils/config.go

```diff
@@ -78,6 +78,7 @@ type factory struct {
 }
 
 type hypervisor struct {
+	Igvm                           string   `toml:"igvm"`
```

Comment:
@wedsonaf what is your thought here? We're explicitly introducing 'igvm' besides initrd, image, kernel, and firmware. since IGVM didn't make it to the community yet and since its name might change, we might think of:
- whether we want to call this IGVM or give it a bit more of an abstract name
- what are the implications here of adding a very CH+MSHV specific field right here - is this fine?